server/grpchelper/status.go

```diff
@@ -103,6 +103,10 @@ func ToStatusError(err error) error {
 	cause := errors.Unwrap(err)
 	if cause == nil {
 		cause = err
+	} else {
+		for errors.Unwrap(cause) != nil {
+			cause = errors.Unwrap(cause)
+		}
 	}
```

Comment:
How about simplifying this code?

```suggestion
	cause = err
	for errors.Unwrap(cause) != nil {
		cause = errors.Unwrap(cause)
	}
```

Reply:
Wow, this is very beautifully simplified code. Thank you!